tooling/minimize/tests/pass/union_marks_bytes.rs

```diff
@@ -0,0 +1,62 @@
+
+#[derive(Clone, Copy)]
+union EnumUnion {
+    data: Option<i32>,
+}
+
+fn test_enum_union(u: EnumUnion) -> i32 {
+    unsafe {
+        match u.data {
+            Some(i) => i,
+            None => 0
+        }
+    }
+}
+
+
+// FIXME: structs cause union to not be well-formed
```

Comment:
What's that about? That should be fixed by now I think.

Reply:
It is